src/services/lnd/service.ts

```diff
@@ -0,0 +1,41 @@
+import { decodeInvoice } from "@domain/ln-invoice"
```

Comment:
if this file is an entry point for the whole service then must be exposed in the index.ts file of this folder